src/mscorlib/model.xml

```diff
@@ -2326,6 +2355,17 @@
       <Member MemberType="Property" Name="InvalidCultureName" />
       <Member MemberType="Property" Name="Message" />
     </Type>
+    <Type Name="System.Globalization.CultureTypes">
+      <Member MemberType="Field" Name="AllCultures" />
```

Comment:
talked offline with Wes and the direction is expose everything. 
Jon, please make sure to expose the whole surface area according to Wes's guidelines. I am seeing you have excluded some APIs in this change so we need to include them.

---
In reply to: [75198460](https://github.com/dotnet/coreclr/pull/6694#discussion_r75198460) [](ancestors = 75198460)